utils/imodels-client-test-utils/src/assertions/BrowserFriendlyAssertions.ts

```diff
@@ -141,13 +143,19 @@ export async function assertNamedVersion(params: {
 export function assertCheckpoint(params: {
   actualCheckpoint: Checkpoint;
   expectedCheckpointProperties: {
-    changesetId: string;
-    changesetIndex: number;
+    changesetId?: string;
+    changesetIndex?: number;
     state: CheckpointState;
   };
 }): void {
-  expect(params.actualCheckpoint.changesetId).to.equal(params.expectedCheckpointProperties.changesetId);
-  expect(params.actualCheckpoint.changesetIndex).to.equal(params.expectedCheckpointProperties.changesetIndex);
+  if (params.expectedCheckpointProperties.changesetId)
```

Comment:
assert that changeset id is not null. it can be empty, but it cannot be null/undefined

Reply:
changesetId can be null if we are acquiring briefcase on empty iModel